shared/package.json

```diff
@@ -38,17 +37,24 @@
     "vendor-install": "node npm-vendor.js",
     "vendor-update": "node npm-vendor.js update",
     "watch-test-file": "KEYBASE_RUN_MODE=devel npm run _helper watch-test-file",
-    "wrap": "npm uninstall fsevents && npm prune && npm dedupe && npm shrinkwrap --dev"
+    "wrap": "npm uninstall fsevents && npm prune && npm dedupe && npm shrinkwrap --dev",
+    "rn-start": "./node_modules/react-native/packager/packager.sh",
+    "rn-gobuild-ios": "./react-native/gobuild.sh ios",
+    "rn-gobuild-android": "./react-native/gobuild.sh android",
+    "rn-packager-wipe-cache": "node ./react-native/wipe-cache.js",
+    "rn-test-ios": "xcodebuild test -project ios/Keybase.xcodeproj -scheme 'Keybase For Test' -destination 'platform=iOS Simulator,name=iPhone 6s,OS=9.3'"
```

Comment:
i'll fix. the testing stuff is pretty broken so i didn't bother. when fixing up the project i killed the test target even